packages/suite/src/components/suite/modals/SafetyChecks/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import { useActions, useSelector } from '@suite-hooks';
+import { RadioButton, H2, P } from '@trezor/components';
+import { Translation, Modal, ModalProps } from '@suite-components';
+import * as deviceSettingsActions from '@settings-actions/deviceSettingsActions';
+import { SafetyCheckLevel } from 'trezor-connect/lib/typescript/trezor/protobuf';
```

Comment:
```suggestion
import { SafetyCheckLevel } from 'trezor-connect';
```

Reply:
Thx. Fixed. This required [one additional change](https://github.com/trezor/connect/compare/d2bbc6a2207aed406be9082d8112a2c25603bfb2..75860f292720ed148a71ec8dde26e464eb3d6e3f) in the Connect PR to export `SafetyCheckLevel` at the root of Connect.

Reply:
on second thought, you dont need to import, and reexport anything.
typescript will resolve this type straight from features so no additional action is required.

just use it like:
`const [level, setLevel] = useState(device.features?.safety_checks);`

Reply:
:+1: Edited (and also reverted the export of `SafetyCheckLevel`.

Pls check. .)